Ship navigation guidance as skills, drop AGENTS.md/CLAUDE.md injection (#184)
alimanfoo/uncoded: .agents/skills/uncoded-doc-navigation/SKILL.md

---
name: uncoded-doc-navigation

> description: Use when finding or reading a codebase's Markdown documentation indexed by uncoded — locating which file and section cover a topic, or orienting to what documentation exists.

Author: ```suggestion
description: Use before searching or reading a codebase's Markdown documentation indexed by uncoded — locating which file and section cover a topic, or orienting to what documentation exists.
```

> description: Navigate a codebase's Markdown documentation indexed by uncoded. Load .uncoded/docs.yaml at session start to see every file and its heading hierarchy, then use Read or grep to reach a specific section.

Author: Mechanical details not relevant here.

Author: Fixed in 38110bb. Now "Use when finding or reading a codebase's Markdown documentation indexed by uncoded — locating which file and section cover a topic, or orienting to what documentation exists." The load-docs.yaml-at-session-start mechanics are dropped from the description and remain in the skill body.

---

# Documentation Navigation

This codebase uses [uncoded](https://github.com/alimanfoo/uncoded) to maintain
a documentation index.

**Step 1 — Orient. Read the docs map first.** Before answering the
user, before any other tool call:

```text
Read .uncoded/docs.yaml
```

`.uncoded/docs.yaml` is an orientation outline: it lists every Markdown file

> and its heading hierarchy. Read it once, in full, at session start.

Author: ```suggestion
`.uncoded/docs.yaml` is an orientation outline: it lists every Markdown file
and its heading hierarchy. Read it in full, now.
```
